aquacomputer_d5next.c

```diff
@@ -211,6 +218,29 @@ static const char *const label_fan_flow_speed_quadro[] = {
 	"Flow speed [l/h]"
 };
 
+struct fan_structure_offsets {
+	u8 voltage;
+	u8 ampere;
```

Comment:
Call this `current` to stay consistent with hwmon.

Reply:
I could not use current as it is already in use by:
./arch/x86/include/asm/current.h:18:17: Fehler: Feld »get_current« als Funktion deklariert
   18 | #define current get_current()

Reply:
Ah I see, just tried it and you're right. `ampere` it is then.